llvm/test/CodeGen/LoongArch/GlobalISel/calllowering-ret.ll

```diff
@@ -0,0 +1,17 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc --mtriple=loongarch32 -global-isel -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=LA32 %s
+; RUN: llc --mtriple=loongarch64 -global-isel -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=LA64 %s
```

Comment:
Do you want to support 32-bit and 64-bit with one GlobalISel instance? I believe only X86 is doing it so far.

Reply:
Thanks for your review.

I’ sorry that I don't quite understand "`with one GlobalISel instance`" you saied. Will this cause any problems in the future? I noticed RISCV did the same.

I'm new to GISel, I will be grateful if you can explain more details for me.